app/models/suggestion.py

```diff
@@ -1,22 +1,27 @@
 from datetime import datetime
 import pytz
+
 from .. import db
+from ..models.resource import Resource, ResourceBase
+
 
-class Suggestion(db.Model):
+class ResourceSuggestion(ResourceBase):
     """
     Association between a resource and potential suggestions for it
     """
-    __tablename__ = 'suggestions'
-    id = db.Column(db.Integer, primary_key=True)
-    resource_id = db.Column(db.Integer, db.ForeignKey('resources.id'))
-    suggestion_text = db.Column(db.String(250))
-    # 0 stands for read, 1 stands for unread
+    resource_id = db.Column(db.Integer, db.ForeignKey(Resource.id))
+    notes = db.Column(db.String(250))
+    # 0 stands for read, 1 stands for unread.
     read = db.Column(db.Boolean, default=False)
     submission_time = db.Column(db.DateTime)
     contact_name = db.Column(db.String(64))
     contact_email = db.Column(db.String(64))
     contact_phone_number = db.Column(db.String(64))
 
+    __mapper_args__ = {
```

Comment:
Haha that doc's "style" conflicts with this doc's "style": http://docs.sqlalchemy.org/en/latest/orm/inheritance.html I think we can pick whatever style we want. Personally, I think the way it is now is easier to read.